apps/api/src/entities/bundle.entity.ts

```diff
@@ -0,0 +1,17 @@
+import { Column, Entity, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
+import { CareActivity } from './careactivity.entity';
+
+@Entity()
+export class Bundle {
+  @PrimaryGeneratedColumn('uuid')
+  bundle_id = '';
+
+  @Column()
+  bundle_name = '';
```

Comment:
Could you modify all the case stylings to `camelCase`

Reply:
Also remove the default assignment. Also, define more column properties like default, type, unique, nullable and things like that

Eg:
```
@Column({type: 'varchar', length: 255, nullable: false, unique: true })
bundleName: string;
```

Reply:
This format should be applied to all the column variables.